balancer-labs/frontend-v2
# Main page loading

# Description

Improves the loading of the main page!

## Type of change


## How should this be tested?

Load up the main investments table on the home page, everything should load much faster but all previous behaviour should be retained.

## Visual context

Speedy load.

## Checklist:
